pcap/gen_defs_windows.go

```diff
@@ -0,0 +1,40 @@
+// +build ignore
+
```

Comment:
add a comment up here detailing what godefs does and that the .bat file should be run to take advantage of this file?

Reply:
done (now in generate_defs.go)